src/services/item/repository.ts

```diff
@@ -287,6 +287,39 @@ export class ItemRepository {
     return farthestItem?.path?.split('.')?.length ?? 0;
   }
 
+  /**
+   * Return all the items where the creator is the given actor.
+   * // TODO: check if it is wanted !
```

Comment:
what's the state of this? Maybe we can have this in an issue and tackle it later.

Reply:
up